options.go

```diff
@@ -108,6 +112,11 @@ func (opt *Options) init() {
 	if opt.IdleCheckFrequency == 0 {
 		opt.IdleCheckFrequency = time.Minute
 	}
+	if opt.MaxBackoffRetryTimeout == 0 {
```

Comment:
Do you want me to change https://github.com/Malwarebytes/redis/blob/a3e42eccb641f8f4bcee2cc1a71ab23593de9564/options.go#L101 and https://github.com/Malwarebytes/redis/blob/a3e42eccb641f8f4bcee2cc1a71ab23593de9564/options.go#L96 too? I was just following the current style of the file. This syntax is used for the other timeouts as well.